### DISCUSSION TITLE
I am using helm chart and looking to enable ENABLE_JAVASCRIPT_CONTROLS, please help to find a place in helm values.yaml to update

### DISCUSSION THREAD

**Discussion by asitkumar13** [apache/superset#24957] [General]
# Allow for javascript controls components
    # this enables programmers to customize certain charts (like the
    # geospatial ones) by inputing javascript in controls. This exposes
    # an XSS security vulnerability
    "ENABLE_JAVASCRIPT_CONTROLS": False,

by default, ENABLE_JAVASCRIPT_CONTROLS is false, and I am looking to update as True. I tried 
  configOverrides:
    "ENABLE_JAVASCRIPT_CONTROLS": "True"
updating in helm values.yaml and deploy but its not working.

**Comment by asitkumar13** [comment]
Hi Team, can you please help me on how can I enable ENABLE_JAVASCRIPT_CONTROLS if I use helm chart to deploy superset.

**Comment by rakeshragipani** [comment]
Hi Team, I tried to update the **SUPERSET_WEBSERVER_JAVASCRIPT_CONTROLS: true** under **configOverrides:**  in **values.yaml**  .
and I updated the helmchart. still, the values are not updated in **config.py** file. Pleae help me with this. Thanks

**Comment by asitkumar13** [comment]
Hi Team, Is there any update on my issue?

**Comment by rashit-saitov** [comment]
Hey, u need to turn on this inside FEATURE_FLAGS. For example:
configOverrides:
  javascript: |
    FEATURE_FLAGS = {
      "ENABLE_JAVASCRIPT_CONTROLS": True
    }
